framework/resources/Functional/semijoin/tpch_sf1Json.json.fail

```diff
@@ -0,0 +1,23 @@
+{
+    "testId": "semiJoinTpchSf1Json",
+    "type": "group",
+    "description": "Test semi join queries for tpch sf1 json data via jdbc",
+    "submit-type": "jdbc",
+    "categories": [
+        "functional"
+    ],
+    "matrices": [
+        {
+            "query-file": ".*.sql",
+            "schema": "dfs.drilltesttpchsf1json",
+            "output-format": "tsv",
+            "expected-file": ".*.e_tsv",
+            "create-data-in-folder": true,
+	    "verification-type": [
+                "in-memory"
+            ]
+        }
+    ],
+    "datasources": [
```

Comment:
Lets copy over data - not good to leave it blank, as we want these to be independent of other suites